common/src/main/java/com/sequenceiq/cloudbreak/auth/ThreadBasedUserCrnProvider.java

```diff
@@ -9,18 +13,15 @@
 
     private static final ThreadLocal<String> USER_CRN = new ThreadLocal<>();
 
+    @Nullable
     public String getUserCrn() {
         return USER_CRN.get();
     }
 
     public String getAccountId() {
-        String userCrn = getUserCrn();
-        if (userCrn != null) {
-            Crn crn = Crn.fromString(userCrn);
-            return crn.getAccountId();
-        } else {
-            throw new IllegalStateException("Crn is not set!");
-        }
+        return ofNullable(
```

Comment:
What do you guys think about this change? For me it looks unnecessary complexity, makes harder to read. If it's okay for others, then it's fine for me too. Lets decide together